File: shortest-root/src/init_db.js
const sqlite3 = require('sqlite3');
const Database = require('better-sqlite3')

const db = new sqlite3.Database('./db.sqlite');
/*
const newDB = Database('newDB.db')
newDB.exec('CREATE TABLE test (id INTEGER PRIMARY KEY AUTOINCREMENT, eventId TEXT NOT NULL);')
newDB.exec(`INSERT INTO test (eventId) VALUES ('woahhhhh')`)
output = newDB.exec('SELECT * FROM test')
console.log(output)
*/
db.run(`CREATE TABLE IF NOT EXISTS events (   
        id INTEGER PRIMARY KEY AUTOINCREMENT,      
        eventId TEXT NOT NULL,   
        UserIds TEXT NOT NULL,  
        HeadId TEXT NOT NULL,
        Modes TEXT NOT NULL,
        Type TEXT NOT NULL,
        Adresses TEXT NOT NULL);`  , function(err){
    if (err){
        console.log(err)
        console.log("what?")
    }
    console.log('JUST WORK ALREADY')

    db.run(`CREATE TABLE IF NOT EXISTS users (
        id INTEGER PRIMARY KEY AUTOINCREMENT,
        Username TEXT NOT NULL);`, function(err){
        if(err){
            console.log(err)
        }
        console.log("cool.")

        db.all("SELECT name FROM sqlite_master WHERE type = 'table' AND name NOT LIKE 'sqlite_%'", function(err, data) {
            if (err){
                console.log(err)
                console.log("ERROR HAS OCCURED")
            }
            else{
                console.log(JSON.stringify(data))
                console.log("ABOVE ME IS THE DATA")
            }
        })
    })  
}) 

File: shortest-root/src/DataBase/init_db.py
import sqlite3
#import requests

connection = sqlite3.connect('database.db')

with open('schema.sql') as f:
    connection.executescript(f.read())

with open('schema2.sql') as f:
    connection.executescript(f.read())

Cursor = connection.cursor()
Cursor.execute("SELECT name FROM sqlite_master WHERE name NOT LIKE 'sqlite_%'")
print(Cursor.fetchall())

#cmd = "INSERT INTO events(EventName, EventType, UserIds, HeadId, Adresses) VALUES (?, ?, ?, ?, ?)"
#Cursor.execute(cmd, ["TYPE", "TESTT", "&", "123", "&"])
#connection.commit()

#Cursor.execute("SELECT * FROM events")
#print(Cursor.fetchone())

connection.commit()
connection.close()
